packages/fileimport-service/Dockerfile

```diff
@@ -1,30 +1,54 @@
-FROM node:16.15-bullseye-slim as node
+FROM node:16.15-bullseye-slim as build-stage
 
-RUN apt-get update && apt-get install -y \
-  python3 \
-  python3-pip \
-  tini \
-  && rm -rf /var/lib/apt/lists/*
-
-ADD https://github.com/ufoscout/docker-compose-wait/releases/download/2.8.0/wait /wait
-RUN chmod +x /wait
+WORKDIR /speckle-server
 
+# add wait
+ENV WAIT_VERSION 2.8.0
+ADD https://github.com/ufoscout/docker-compose-wait/releases/download/${WAIT_VERSION}/wait ./wait
+RUN chmod +x ./wait
+
+# Add tini
+ENV TINI_VERSION v0.19.0
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini ./tini
+RUN chmod +x ./tini
+
+# Add python virtual env
+WORKDIR /venv
+RUN apt-get update && \
+    DEBIAN_FRONTEND=noninteractive apt-get install \
+    --no-install-suggests --no-install-recommends --yes \
+    python3-venv=3.9.2-3 && \
+    python3 -m venv /venv
+
+# pip install
+COPY packages/fileimport-service/requirements.txt /requirements.txt
+RUN /venv/bin/pip install --disable-pip-version-check --requirement /requirements.txt
+
+# yarn install
+WORKDIR /speckle-server
 ARG NODE_ENV=production
 ENV NODE_ENV=${NODE_ENV}
-
-WORKDIR /speckle-server
-
 COPY .yarnrc.yml .
 COPY .yarn ./.yarn
 COPY package.json yarn.lock ./
 
 WORKDIR /speckle-server/packages/fileimport-service
-COPY packages/fileimport-service/package.json ./
+COPY packages/fileimport-service/package.json .
 RUN yarn workspaces focus --production
 
-COPY packages/fileimport-service/requirements.txt ./
-RUN pip install -r requirements.txt
+FROM gcr.io/distroless/nodejs:16 as production-stage
 
+WORKDIR /speckle-server/packages/fileimport-service
 COPY packages/fileimport-service .
-
-CMD ["yarn", "node", "src/daemon.js"]
+COPY --from=build-stage /speckle-server/wait /wait
+COPY --from=build-stage /speckle-server/tini /tini
+COPY --from=build-stage /venv /venv
+COPY --from=build-stage /speckle-server/node_modules ./node_modules
+
+# HACK
+# the node script calls 'python3', which works when running locally.
+# Force use of the python virtual environment in the image,
+# as this has the pip requirements installed.
+ENV python3="/venv/bin/python3"
```

Comment:
This should use the proper environment variables to set the python environment

Reply:
![image](https://user-images.githubusercontent.com/57442769/183454868-0a4465cc-9e7d-49bc-a07e-87bad2937ad4.png)

The way the python venv activation script does it, is actually prefixing the `PATH` env variable, with the path, where the venv python binary lives. Its a good non destructive solution.
Probably the `VIRTUAL_ENV` variable does not need to be set, lets test without that.